src/System.IO.FileSystem.Watcher.Polling/tests/project.json

```diff
@@ -1,12 +1,22 @@
 {
+  "name": "System.IO.FileSystem.Watcher.Polling.Tests",
   "dependencies": {
-    "Microsoft.NETCore" : "5.0.0",
-    "System.Console":  "4.0.0-*",
-    "System.Threading.Thread": "4.0.0-*",
-    "System.Diagnostics.TraceSource": "4.0.0-*",
-    "xunit": "2.0.0-*"
+    "Microsoft.NETCore.TestHost": "1.0.0-beta-23409",
+    "Microsoft.NETCore.Platforms": "1.0.1-beta-23409",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.1-beta-23409",
+    "src": {
+          "type": "build",
+          "version": "1.0.0"
+        },
+    "Microsoft.NETCore": "5.0.0",
+    "System.Console": "4.0.0-beta-*",
```

Comment:
@KrzysztofCwalina I tried it for a few projects, but a couple had issue where the dotnetstandard had conflicts with some of their own dependencies. I had to switch to this to make it work and then I just copied/pasted the corehost dependencies across the test projects.